### DISCUSSION TITLE
iDawarich iOS (no Tracking only for viewing data)

### DISCUSSION THREAD

**Discussion by cutzenfriend** [Freika/dawarich#442] [Show and tell]
Hey there,

The Webinterface is great no questions asked. But on mobile I just wanted something more snappy. So I built myself a little iOS app to see my geopoints as a route on a map and the overall stats.

<img src="https://github.com/user-attachments/assets/ace12e63-5520-44dd-8ff2-6b29cbb77893" width="300px">

<img src="https://github.com/user-attachments/assets/2ee55651-0a5f-46d8-8cd5-06df0b599e42" width="300px">

<img src="https://github.com/user-attachments/assets/e44a41f5-e299-4a4c-b9de-035a2b62032c" width="300px">

<img src="https://github.com/user-attachments/assets/f9fcf842-3a9b-4cd5-bc00-c370febe4283" width="300px">

I also added a little caching. Nothing fancy just for a quick check. It's completely written in swift and uses the official Dawarich API. Currently I just have it installed on my device as a developer build. Are you guys interested? Should I publish it in the app store?

**Comment by sunstep** [comment]
I wanted to keep it secret while it was in development, but I have a project in the works for Dawarich. It will be a mobile app written in Dart/ Flutter framework. This means the app will be supporting both Android/ IOS. 

The app I am making is supposed to align much as possible with the features of the actual Dawarich. Besides that, the app is also supposed to provide tracking functionality, so it is no longer necessary to use third party apps on your phone to track your location. All functionality should be available within the Dawarich eco-system. 

Features that are currently implemented/ to be implemented:
- Logging in with your Dawarich account
- A map showing places where you have been
- Page displaying your stats
- A page for managing your points
- Tracking functionality, which is supposed to be configurable for the user, so the user gets to decide how much battery the app uses.
- User management
- Settings to manage this app, but might also contain management for dawarich settings (through api).
- Offline functionality

As the app gets more stable, I will reveal more. 

It is very expensive to publish an app to the appstore. So please take this comment into consideration whether it is worth it or not for you.

This comment is not meant to stop you from publishing your app, but I know Apple is a horrible company for developers. They charge you for publishing apps, they take money from you if you make any revenue etc. So it is up to you to consider if publishing your app is worth it or not. I am just telling you that I have my own version of a Dawarich app in the works and it will be packed with more features and will cover both platforms.

**Reply by pablogglz** [reply]
I'd love it. Regarding costs, can't you publish it on F-Droid? I have no idea

**Reply by sunstep** [reply]
Publishing on Android is relatively cheap compared to publishing on IOS. Android will not be a problem at all. I can already tell that I will publish the app on Android first when it is stable. I don't know yet what I will do regarding the costs of IOS but it is most likely that I will just have to pay up at the end of the day, to make the app accessible to IOS users.

**Comment by Freika** [comment]
@cutzenfriend resized the images in your original post so they wouldn't look huge on the desktop screen 😉 
The app looks great!

**Comment by tismofied** [comment]
@cutzenfriend Did you release it? I'd love to give it a try.

**Reply by Freika** [reply]
Can't say for the app from the main message, but we just released our own yesterday 🎉 

https://apps.apple.com/de/app/dawarich/id6739544999?l=en-GB

**Reply by tismofied** [reply]
Awesome news. Thank you @Freika
